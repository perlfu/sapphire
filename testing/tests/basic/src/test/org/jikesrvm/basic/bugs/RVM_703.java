/*
 *  This file is part of the Jikes RVM project (http://jikesrvm.org).
 *
 *  This file is licensed to You under the Eclipse Public License (EPL);
 *  You may not use this file except in compliance with the License. You
 *  may obtain a copy of the License at
 *
 *      http://www.opensource.org/licenses/eclipse-1.0.php
 *
 *  See the COPYRIGHT.txt file distributed with this work for information
 *  regarding copyright ownership.
 */
package test.org.jikesrvm.basic.bugs;

public class RVM_703 {

  public static void main(String[] a) {
    foo();
  }

  private static void foo() {
    bar();
  }

  private static void bar() {
    Throwable t = new Throwable();
    for (StackTraceElement l : t.getStackTrace()) {
      System.out.println(l.getClassName() + " " + l.getMethodName());
    }
  }
}
