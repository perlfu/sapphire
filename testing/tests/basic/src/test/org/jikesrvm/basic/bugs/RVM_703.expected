test.org.jikesrvm.basic.bugs.RVM_703 bar
test.org.jikesrvm.basic.bugs.RVM_703 foo
test.org.jikesrvm.basic.bugs.RVM_703 main
